<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMusicWindow</class>
 <widget class="QMainWindow" name="MyMusicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
	background:rgba(255,255,255,100%);
	border-radius:20px;
	background-color: rgb(140, 140, 140);
}
#pushButton{
	background:#1e1d1d;
	border-radius:15px;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	color: rgb(200,200,200); 
	background:rgba(20,20,20,90%)
}

#MyMusicWindow{
border-radius:15px;
}
#widget{
	border-radius:15px;
	background:rgba(70,70,70,80%)
}
QLineEdit{
	border-radius:15px;
	background:#1e1d1d;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	background:rgba(40,40,40,80%);
	color: rgb(200,200,200); 
	padding-left: 15px;
}
QLineEdit:hover
{
	color: rgb(255,255,255); 
    background-color: rgb(45,45,45);
	border-style: solid;
	border-width: 2px;
	border-color: rgb(20,20,20);
}
#widget_2{
	border-radius:15px;
	background:rgba(70,70,70,80%)
}
#widget_3{
	border-radius:15px;
	background:rgba(70,70,70,80%)
}
#search{
	background:#1e1d1d;
	border-radius:15px;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	color: rgb(200,200,200); 
	background:rgba(40,40,40,80%)

}
#search:hover{
	color: rgb(255,255,255); 
    background-color: rgb(45,45,45);
	border-style: solid;
	border-width: 2px;
	border-color: rgb(20,20,20);
}
#search:pressed {
	background:#1e1d1d;
	border-radius:15px;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	color: rgb(255,255,255); 
	background:rgba(20,20,20,90%)
}

QTableView{  
	background-color: rgba(70,70,70,80%);
    border: none;
    selection-background-color: rgba(80, 80, 80,80%);
	font-size: 25px;
	font-family: Microsoft YaHei ;
}
QHeaderView::section{         /*表头设置*/
    height:50px;    			/*表头高度*/
    color: white;
    font-size: 30px;
    font-weight: 900;
	font-family: Microsoft YaHei ;
    background-color: rgba(130, 130, 130,80%);
    text-align: center;
	border: 1px solid rgb(100,100,100);
}

QTableView::item:!alternate:!focus{
    background-color: rgba(200, 200, 200,80%);    /*交替行的另一颜色*/
    selection-color: rgba(0, 0, 0,80%);
	border: 1px solid rgb(100,100,100);
}
QTableView::item:alternate:!focus{
    background-color: rgba(130, 130, 130,80%);
	selection-color: rgba(0, 0, 0,80%);
	border: 1px solid rgb(100,100,100);
}

QScrollBar:vertical{ 
	width:20px;
	background-color:rgba(150, 150, 150,80%);/*垂直滚动条 背景 颜色*/
	margin:1px ;
	border:1px solid rgb(100,100,100);/*边框颜色*/
	padding-top:22px;
	padding-bottom:22px;
}

QScrollBar::handle:vertical{
 
	background:rgba(200, 200, 200,80%);
	min-height:50px;
 	border:1px solid rgb(255,255,255)
}

QScrollBar::add-page:vertical{
 	background:rgba(100, 100, 100,80%);
	min-height:50px;
 	border:1px solid rgb(100,100,100)
}


QScrollBar::sub-page:vertical{
	background:rgba(100, 100, 100,80%);
	min-height:50px;
 	border:1px solid rgb(100,100,100)
 
}

#play_menu{
	border-radius:15px;
	background:rgba(60,60,60,80%)
}

QSlider::groove:horizontal {
border: 0px solid #bbb;
}

QSlider::sub-page:horizontal {
	background: rgb(100,100,100,80%);
	border: 0px solid #777;
	border-radius: 2px;
	margin-top:15px;
	margin-bottom:15px;
}

QSlider::add-page:horizontal {
	background: rgb(200,200,200,80%);
	border: 0px solid #777;
	border-radius: 2px;
	margin-top:15px;
	margin-bottom:15px;
}

QSlider::handle:horizontal {
 	width: 10px;
 	background-color: #585858;
 	margin-top:10px;
 	margin-bottom:10px;
	border-radius: 100px; 
	border: 1px solid rgb(150,150,150);
}
QSlider::handle:horizontal:hover {
 	width: 15px;
 	background-color: #585858;
 	margin-top:5px;
 	margin-bottom:5px;
	border-radius: 100px; 
	border: 1px solid rgb(150,150,150);
 
}
#label{
	border-radius: 25px; 
}




</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>110</y>
      <width>601</width>
      <height>481</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2{
	background:#1e1d1d;
	border-radius:15px;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	color: rgb(200,200,200); 
	background:rgba(20,20,20,90%)
}</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>861</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QPushButton" name="front_music">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#front_music{
	image: url(:/png/front_normal.png);
	border-radius:30px;
}
#front_music:hover{

	image: url(:/png/front_hover.png);
	border-radius:30px;
}
#front_music:pressed{
	image: url(:/png/front_pressed.png);
	border-radius:30px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="next_music">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#next_music{
	image: url(:/png/next_normal.png);
	border-radius:30px;
}
#next_music:hover{
	image: url(:/png/next_hover.png);
	border-radius:30px;
}
#next_music:pressed{
	image: url(:/png/next_pressed.png);
	border-radius:30px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="play_button">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#play_button{
	image: url(:/png/play_normal.png);
	border-radius:30px;
}
#play_button:hover{
	image: url(:/png/play_hover.png);
	border-radius:30px;
}
#play_button:pressed{
	image: url(:/png/play_pressed.png);
	border-radius:30px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>309</x>
       <y>70</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(253, 253, 253);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>70</y>
       <width>64</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(249, 249, 249);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="cover_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="singgername">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(247, 247, 247);</string>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QLabel" name="songname">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(249, 249, 249);</string>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_5{
	image: url(:/png/coll_normal1.png);
	border-radius:30px;
}
#pushButton_5:hover{
	
	image: url(:/png/coll_hover.png);
	border-radius:30px;
}
#pushButton_5:pressed{
	image: url(:/png/colled.png);
	border-radius:30px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>221</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QPushButton" name="like_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#like_button{
	border-radius:15px;
	background:#1e1d1d;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	background:rgba(40,40,40,80%);
	color: rgb(200,200,200); 
	padding-left: 15px;
}
#like_button:hover
{
	color: rgb(255,255,255); 
    background-color: rgb(45,45,45);
	border-style: solid;
	border-width: 2px;
	border-color: rgb(20,20,20);
}</string>
     </property>
     <property name="text">
      <string>我喜欢的音乐 </string>
     </property>
    </widget>
    <widget class="QPushButton" name="reco_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#reco_button{
	border-radius:15px;
	background:#1e1d1d;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	background:rgba(40,40,40,80%);
	color: rgb(200,200,200); 
	padding-left: 15px;
}

#reco_button:hover
{
	color: rgb(255,255,255); 
    background-color: rgb(45,45,45);
	border-style: solid;
	border-width: 2px;
	border-color: rgb(20,20,20);
}</string>
     </property>
     <property name="text">
      <string>今日推荐  </string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>40</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入歌名或者歌手</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{
	background:#1e1d1d;
	border-radius:15px;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	color: rgb(200,200,200); 
	background:rgba(20,20,20,90%)

}</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_4{
	background:#1e1d1d;
	border-radius:15px;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	color: rgb(200,200,200); 
	background:rgba(20,20,20,90%)
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_3{
	background:#1e1d1d;
	border-radius:15px;
	font-size: 24px ;
	font-family: Microsoft YaHei ;
	color: rgb(200,200,200); 
	background:rgba(20,20,20,90%)
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
